cas-server-webapp-support/src/main/java/org/jasig/cas/web/support/InspektrThrottledSubmissionByIpAddressAndUsernameHandlerInterceptorAdapter.java

```diff
@@ -148,6 +148,6 @@ public final void setAuthenticationFailureCode(final String authenticationFailur
      */
     protected String constructUsername(final HttpServletRequest request, final String usernameParameter) {
         final String username = request.getParameter(usernameParameter);
-        return "[username: " + (username != null ? username : "") + ']';
+        return (username != null ? username : "unknown") + "+password";
```

Comment:
No this wont work. We need to make sure only the username is actually recorded, so this method here can do its job. You wont always have username+password, because you may have a different authN handler that does nothing with the password.